src/modules/lua_mtev_crypto.c

```diff
@@ -435,8 +435,11 @@ mtev_lua_crypto_rsa_gencsr(lua_State *L) {
   if (!X509_REQ_set_version(req,0L)) /* version 1 */
     REQERR("crypto.rsa:gencsr could not set request version");
 
-  X509V3_CTX ext_ctx;
   lua_getfield(L,-1,"addext");
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
+  if(!lua_isnil(L,-1)) REQERR("addext requires OpenSSL 1.1 or greater");
```

Comment:
I don't think so, because I allow in the code for setext to be omitted (and error if it is set to inform the user), but someone would have to confirm by trying it
Do you have a way to test?

Reply:
notice it says Lua_isnil, so if setext is nil then no error, if setext is set to something then error